src/problem1.rb

```diff
@@ -0,0 +1,12 @@
+
+# Given an integer array of size N,
+# compute the sum of all even numbers in this array
+class Problem1
+  # @param [Array] numbers
+  def self.sum_of_even_numbers(numbers)
+    return 0 if numbers.nil? || numbers.empty?
+    sum = 0
+    numbers.each { |x| sum += x.even? ? x : 0 }
+    sum
```

Comment:
Good. But can be better. Can you write it as 1 liner ?

Reply:
Yep

Reply:
One-liner is pretty new to me, but for now I don't see it very clean...

Reply:
Ok, so I managed to do something shorter and cleaner